tests/e2e/csi_static_provisioning_basic.go

```diff
@@ -356,6 +356,82 @@ var _ = ginkgo.Describe("Basic Static Provisioning", func() {
 		framework.ExpectNoError(fpv.WaitForPersistentVolumeDeleted(client, pv.Name, poll, pollTimeout))
 		pv = nil
 	})
+
+	/*
+		This test verifies the static provisioning workflow.
+
+		Test Steps:
+		1. Create FCD and wait for fcd to allow syncing with pandora.
+		2. Create PV Spec with volumeID set to FCDID created in Step-1, and PersistentVolumeReclaimPolicy is set to Delete.
+		3. Create PVC with the storage request set to PV's storage capacity.
+		4. Wait for PV and PVC to bound.
+		5. Create a POD.
+		6. Verify volume is attached to the node and volume is accessible in the pod.
+		7. Verify container volume metadata is present in CNS cache.
+		8. Delete POD.
+		9. Verify volume is detached from the node.
+		10. Delete PVC.
+		11. Verify PV is deleted automatically.
+	*/
+	ginkgo.It("[csi-block-vanilla] Verify static provisioning workflow using same PV name", func() {
+		var err error
+
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		ginkgo.By("Creating FCD Disk")
+		fcdID, err := e2eVSphere.createFCD(ctx, "BasicStaticFCD", diskSizeInMb, defaultDatastore.Reference())
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		deleteFCDRequired = false
```

Comment:
deleteFCDRequired  should be true here. 
If the test case fails in between then Deletion of FCD is handled in After each block.